articles/container-registry/container-registry-faq.md

```diff
@@ -87,6 +88,11 @@ The error is seen when the user has permissions on a registry but doesn't have R
 az role assignment create --role "Reader" --assignee user@contoso.com --scope /subscriptions/<subscription_id> 
 ```
 
+### Firewall rules are updated successfully but they do not take effect
+
+It takes some time to propagate firewall rule changes. After you change firewall settings, please wait for several minutes before verifying this change.
+
```

Comment:
After you change firewall settings, please wait for a few minutes before verifying this change.